packages/sdks/typescript/src/index.ts

```diff
@@ -195,26 +201,27 @@ export class Latitude {
     }
 
     return this.handleStream({
-      stream: response.body!,
+      body: response.body! as Readable,
       onEvent,
       onFinished,
       onError,
     })
   }
 
   private async handleStream({
-    stream,
+    body,
     onEvent,
     onFinished,
     onError,
   }: StreamResponseCallbacks & {
-    stream: ReadableStream
+    body: Readable
   }) {
     let conversation: Message[] = []
     let uuid: string | undefined
     let chainResponse: ChainCallResponseDto | undefined
 
     const parser = new EventSourceParserStream()
+    const stream = nodeFetchResponseToReadableStream(body)
```

Comment:
Node fetch doesn't play well directly with `ReadableStream` web streams. I did this helper function to convert node-fetch events to a readable stream. What do you think?